frontend/app/controllers/Giraffe.scala

```diff
@@ -28,7 +28,11 @@ object Giraffe extends Controller {
 
 
   val chargeId = "charge_id"
-  val maxAmount: Option[Int] = 2000.some
+
+  def maxAmount(currency: Currency): Option[Int] = currency match {
+    case CountryGroup.Australia.currency => 3500.some
```

Comment:
we still need to confirm what the actual max amount should be in australian dollars